packages/components-css/icon-css/src/_mixin.scss

```
@@ -0,0 +1,21 @@
+@mixin nl-icon {
+  block-size: var(--nl-icon-block-size, 1em);
+  color: var(--nl-icon-color, currentcolor);
+  display: inline-block;
+  font-size: var(--nl-icon-font-size, 1em);
+  inline-size: var(--nl-icon-inline-size, 1em);
+  inset-block-start: var(--nl-icon-inset-block-start, 0);
+  pointer-events: none;
+  position: relative;
+}
+
+@mixin nl-icon__svg {
+  /* stylelint-disable-next-line property-disallowed-list */
+  height: 100%;
+  /* stylelint-disable-next-line property-disallowed-list */
+  width: 100%;
+}
+
+@mixin nl-icon--dir-aware {
```

`dir-aware` is niet echt een term die voorkomt in web standards, ik zou het liefst wel iets in die richting doen. In Unicode heet de property `Bidi_Mirrored`, misschien is dat iets? Ik had niet het idee dat er in CSS al een API naam is die heel erg geschikt is voor ons doeleinde.

```suggestion
@mixin nl-icon--bidi-mirrored {
```

---

De oplossing met `scaleX()` is niet gebaseerd op CSS logical properties, dus misschien moeten we documenteren (en een issue aanmaken voor tech debt?) dat als CSS hier een mogelijkheid voor biedt dat we een meer geschikte oplossing moeten toepassen, om [`writing-mode: vertical-rl` te ondersteunen](https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode).